src/support/ahrefs-client.js

```diff
@@ -88,4 +88,35 @@ export default class AhrefsAPIClient {
 
     return this.sendRequest('/site-explorer/broken-backlinks', queryParams);
   }
+
+  async getOrganicKeywords(url) {
+    const formatDate = (date) => {
+      const year = date.getFullYear();
+      const month = String(date.getMonth() + 1).padStart(2, '0');
+      const day = String(date.getDate()).padStart(2, '0');
+
+      return `${year}-${month}-${day}`;
+    };
+    const today = new Date();
+
+    const queryParams = {
+      country: 'au',
```

Comment:
Is `au` country applicable to all sites?

Reply:
no, this is site specific. @alinarublea just showed me how I can integrate the alert config, I will integrate it there